docs/source/en/core/development.md

```diff
@@ -1 +1,339 @@
-this document is still waiting for translation, see [Chinese Version](/zh-cn/core/development.html)
\ No newline at end of file
+title: Local Development
+---
+
+To improve development experience, convenient ways to develop in local environment, debug and run unit testing are provided in egg. 
```

Comment:
We provide convenient ways for development, debugging, and unit tests to improve your development experience.